content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,14 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns. 
+This assembly of multiple machine learning methods is called ensemble learning. In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
+Sometimes a consensus approach is ideal to select predictive features from a group of independently running models, other times one may coordinate various learning methods in such a way that one model uses information from another model in augmenting its predictive capability. 
+Random forest algorithms are a great example of ensemble learning where a consensus of the features selected through many different and independent decision trees are considered and ranked to make a final ranked list of important features that are predictive of a certain outcome (e.g. disease) [@doi:10.1186/1472-6947-13-134, @doi:10.1023/A:1010933404324].
+However, successful application of consensus based ensemble learning algorithms prefer what is known as "gold standard" data. These generally refer to well labeled cases where the diagnosis or label of a data point in the training dataset has very little uncertainty (or "noise") associated with it [@doi:10.1093/jamia/ocw028]. 
+In most cases of rare disease, due to the inherent nature of being less defined, the symptoms as well as any underlying biology comes with a reasonable amount of uncertainty (or "noise") leading to a "silver standard" dataset[@doi:10.1109/TNNLS.2013.2292894, @doi:10.1093/jamia/ocw028].
```

Comment:
I have never heard of silver dataset, interesting! I did recently hear someone referring to bronze-standard methods :) 

My experience is that rare diseases lack a good gold standard not due to noise, but rather your initial point in this section (a paucity of data). I'm also not sure if "noise" is the right way to describe phenotypic heterogeneity/uncertainty. In the case of complex diseases, the phenotypes are genuinely variable, not meaningless (and I tend to think of noise as meaningless data - not true variance).

Reply:
I see what you mean. Actually I thought the same until I came across this paper(https://pubmed.ncbi.nlm.nih.gov/24808033/). It describes various kinds of noise in the context of machine learning like this:
"... anything that obscures the relationship between the features of an instance and its class. .... noise is also described as consisting of nonsystematic errors ......... In the literature, two types of noise are distinguished: feature(or attribute) and class noises. On the one hand, feature noise affects the observed values of the feature, e.g. by adding a small Gaussian noise to each feature during measurement. On the other hand, class noise alters the observed labels assigned to instances, e.g. by incorrectly setting a negative label on a positive instance in binary classification." 

In our case the term "label-noise" may be more appropriate in describing the true phenotypic uncertainty in the features that seem close to the "Noisy not at random" (NNAR) model described in the paper.